content/doc/developer/publishing/releasing-experimental-updates.adoc

```diff
@@ -2,9 +2,19 @@
 title: Publishing Experimental Plugin Releases
 layout: developer
 ---
+The link:https://github.com/jenkinsci/plugin-installation-manager-tool[Plugin Manager tool] downloads plugins and their dependencies into a folder, which enables easier import into an instance of Jenkins.
+This benefits the user by providing more information about the plugins being downloaded, such as available updates or security warnings and many more.
 
-To simplify delivery of beta versions of plugins to interested users, the Jenkins project publishes an _experimental update center_.
-It includes alpha and beta versions of plugins, which aren't usually included in the regular update center.
+To simplify delivery, link:/doc/developer/plugin-development/incrementals/[Incremental] builds can be used. 
+This provides the user with flexibility to create incremental versions of Jenkins core or an upstream plugin on which under development downstream plugin can depend upon. 
+This also provides the ability to  switch between and keep track of incremental versions.
+The plugin installation manager tool with an incremental build is a preferred technique to test and explore unreleased plugin versions. It is much easier for a Jenkins administrator to add the experimental update center to their Jenkins controller. 
```

Comment:
```suggestion
This also provides the ability to switch between and keep track of incremental versions.
The plugin installation manager tool, with an incremental build, is a proven technique to test and explore unreleased plugin versions. 
These abilities make it easier for a Jenkins administrator to add the experimental update center to their Jenkins controller. 
```
suggestion to fix some spacing, update the sentence per line formatting, and suggestion for rewording the last sentence to be a bit more descriptive.